### ISSUE TITLE
packed items with different than fixed amounts

### ISSUE THREAD

**Issue by litzvi** [litzvi/avc-beta#577]
perhaps change the bound  of amount in packed items, so can display a smaller bag together.

**Comment by litzvi** [comment]
Might help with QC problem too

**Comment by mei309** [comment]
it will loose all the meaning of the separation

**Comment by litzvi** [comment]
Yes, won't have packed, instead will have a default unit amount that will be filled as the unit, but can be changed.
This way if they have another part of a bag they packed, can be shown as another bag but with a reduced weight.
